### ISSUE TITLE
Missing pickled files for hate speech detection

### ISSUE THREAD

**Issue by gak97** [hate-alert/HateMM#1]
Hi,

Thank you for creating this dataset and the models. I see that there are pickled files mentioned in all the feature extraction files and the model training files. However, these pickled files neither are uploaded nor the way to generate them has been explained. As a result, it's not possible to run the code files without running into the below error. 

`Traceback (most recent call last):
  File "/home/kumar/workspace/HateMM/Codes/2.BERTandHateXPlainEmbedding.py", line 17, in <module>
    with open(FOLDER_NAME+'all__video_vosk_audioMap.p','rb') as fp:
FileNotFoundError: [Errno 2] No such file or directory: './Dataset/hate_videos/all__video_vosk_audioMap.p'`

Could you please explain how to resolve this? Also, you mentioned in the paper that you have used the Vosk tool to obtain the transcripts. Could you please provide the transcripts or the code to obtain those transcripts?

**Comment by Chunyu-Hugh** [comment]
yes, if we don't have those files we cannot run the model. meanwhile there are some files we need to run the code.
